designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,263 @@
+- Repo: No ESLint repos, only the ones mentioned below (owned by private
+  persons)
+- Start Date: 2022-05-31
+- RFC PR:
+- Authors: [@aladdin-add](https://github.com/aladdin-add),
+  [@MichaelDeBoey](https://github.com/MichaelDeBoey),
+  [@ota-meshi](https://github.com/ota-meshi) &
+  [@voxpelli](https://github.com/voxpelli)
+
+# `@eslint-community` GitHub organisation
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+Facilitate a GitHub organisation
+([@eslint-community](https://github.com/eslint-community)) where community
+members could maintain widely used ESLint related packages that would otherwise
+get abandoned.
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+There a some widely used ESLint related packages that aren't actively maintained
+anymore and we'd like to transfer them to a community owned GitHub organization
+(`@eslint-community`), so we can give them back to the community.
+
+## Detailed Design
+
+<!--
+   This is the bulk of the RFC.
+
+   Explain the design with enough detail that someone familiar with ESLint
+   can implement it by reading this document. Please get into specifics
+   of your approach, corner cases, and examples of how the change will be
+   used. Be sure to define any new terms in this section.
+-->
+
+### How do you envision `@eslint-community`?
+
+Just like the [@jest-community](https://github.com/jest-community) org, the
+`@eslint-community` org would be a collection of important/widely used ESLint
+plugins/packages that aren't maintained anymore and are given back to the
+community.
+
+> Community repos for ESLint related projects
+
+The projects could be released under the `@eslint-community` scope in a later
+stage.
+
+The benefit of having the org in place would be that we could create a structure
+to streamline the packages more, i.e. tools used, supported Node/ESLint
+versions, ...
+
+### Which projects should be involved?
+
+I think all of [@aladdin-add](https://github.com/aladdin-add),
+[@ota-meshi](https://github.com/ota-meshi) &
+[@voxpelli](https://github.com/voxpelli) & myself would be in for helping to
+maintain at least the following packages:
+[`@mysticatea/eslint-plugin`](https://github.com/mysticatea/eslint-plugin),
+[`eslint-plugin-eslint-comments`](https://github.com/mysticatea/eslint-plugin-eslint-comments),
+[`eslint-plugin-es`](https://github.com/mysticatea/eslint-plugin-es),
+[`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node),
+[`eslint-utils`](https://github.com/mysticatea/eslint-utils) (all from
```

Comment:
If we need to have `eslint-utils` in this community, we probably need to add `regexpp` to this community as well.
`regexpp` and `eslint-utils` are also used in the eslint core.
https://github.com/mysticatea/regexpp
https://github.com/mysticatea/eslint-utils

Reply:
Or should the packages used in the eslint core be separate from the organization proposed in this RFC?

Reply:
@ota-meshi Maybe not needed to decide now. Would make sense for ESLint itself to maybe take them on.

Reply:
Since `regexpp` is used in the main repo, I think it should (at least) be in the new org as well.
Since it's extracted into a personal account, I don't think the core team wants to maintain the package themselves.